### DISCUSSION TITLE
Setting up Bootstrap on local machine

### DISCUSSION THREAD

**Discussion by xarcode** [twbs/bootstrap#38127] [Q&A] [answered]
Hey all, I am trying to make some changes to the Bootstrap docs as pointed out by this issue [https://github.com/twbs/bootstrap/issues/38122](url). How do I set up my local machine properly, could anyone point to the docs which has a good guide for the setup process or maybe someone can offer help in doing so? Thanks in advance !

**Comment by julien-deramond** [comment, marked as answer]
Hey @xarcode 

You can start by the ["Pull requests"](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md#pull-requests) section of our Contributing Guidelines to create your fork, your branch and later your PR.
To compile and launch the documentation locally, you can do the following:
- `npm i`
- `npm run start`
- Go to http://localhost:9001/ to see the docs

**Reply by xarcode** [reply]
Got it! Thanks!
